### PR TITLE
Download logs as .txt file

### PR DESCRIPTION
**Pull request by gojeda-eb**
Now, it is possible to download all logs from a match at "Match Details" section. The button "Download as txt" has been added for such end.
Captura de Pantalla 2022-10-12 a la(s) 17 26 24

### CODE REVIEW

File: development/common/match_result_text.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+CELL_SIZE = 5
+LARGE = 17
+ROW_SIZE = LARGE * CELL_SIZE
+'9-17, 21-34, 38-46, 52-56'
+
+
+def add_text_if_data(currentString, data, index):
+    newStr = currentString
+    newStr += "---------------------------- Player Command -------------------------------------\\n"
+    #  myStr += "Player: " + data[index]["current_player"] + "\\n" # "current_player" is bugged...
```
Comment (facosta-eb) [inline development/common/match_result_text.py:12]: Remove commented code

File: development/tests/test_match_result_text.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+from django.test import TestCase
+
+from development.common.match_result_text import generate_text
+
+
+class TestMatchResultText(TestCase):
+
+    '''
+    'action': 'SHOOT', 'data': '{"from_row": 8, "from_col": 0, "direction": "NORTH",
```
Comment (facosta-eb) [inline development/tests/test_match_result_text.py:11]: Is this used in any way? if not it should be removed

File: development/views.py [hunk 3]
```diff
@@ -11,8 +11,10 @@
     get_matches_of_connected_user,
     get_matches_results,
 )
+from .common.match_result_text import generate_text
 from development.forms import ChallengeForm
 from development.server_requests import (
+    # get_all_logs,
```
Comment (facosta-eb) [inline development/views.py:17]: same

File: development/tests/test_match_result_text.py (hunk 2 above)
Comment (gojeda-eb) [reply development/tests/test_match_result_text.py:11]: Working on it. I left it by accident, sorry.

File: development/tests/test_match_result_text.py [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+            "from_col": "0",
+            "direction": "NORTH",
+            "game_id": "77491184-4bbf-11ed-9308-0242ac120005",
+            "turn_token": "d2ab9417-6301-430b-b536-8d6b6be5e21b",
+        },
+        "current_player": "player1"
+    }
+
+    '''
+    {'player_1': 'VirtualEnv', 'score_1': '1000', 'remaining_turns': '199',
```
[the remark this answers is not in the document]
Comment (gojeda-eb) [reply development/tests/test_match_result_text.py:30]: Working on it

File: development/views.py (hunk 3 above)
Comment (gojeda-eb) [reply development/views.py:17]: On it.

**Review by facosta-eb — APPROVED** [review]

**Pull request merged**